apps/simple-server-example/src/client/client.tsx

```diff
@@ -0,0 +1,92 @@
+import { useMultiplayerSync } from '@tldraw/sync'
+import { useCallback } from 'react'
+import { createRoot } from 'react-dom/client'
+import {
+	Route,
+	RouterProvider,
+	createBrowserRouter,
+	createRoutesFromElements,
+	useParams,
+} from 'react-router-dom'
+import { AssetRecordType, Editor, TLAsset, TLAssetStore, Tldraw, getHashForString } from 'tldraw'
+import 'tldraw/tldraw.css'
+
+const assets: TLAssetStore = {
```

Comment:
i'd maybe consider moving `asset` + `unfurl` below `App` in this file. I feel like from a "reading this example for the first time" pov you want `App` up front to get the big picture of what the parts are and how they fit together, then continue reading to delve into the implementation details of assets/url unfurling

Reply:
I think I'm just gonna copy your client code and hard code the urls